### ISSUE TITLE
When training with the default config on data_tf, I still get intersecting mesh.

### ISSUE THREAD

**Issue by midskavid** [noahcao/Pixel2Mesh#3]
Despite the overall shape of the mesh is as desired, the mesh folds and has self intersections to achieve that. should I be using a different options file? What am I doing wrong? Any help would be appreciated!

**Comment by ultmaster** [comment]
I didn't quite get you. Do you mean you have intersecting meshes when evaluation? Pixel2Mesh doesn't guarantee that there is no intersection. It discourages intersection, instead of avoiding.

**Comment by midskavid** [comment]
No, not during evaluation.. but training.. I was saving the generated objs during training and they had self intersections..  Does that improve with a lot of epochs or am I weighing some parameters wrong?

**Comment by ultmaster** [comment]
Yes. You will expect a lot of epochs before you see something meaningful. You can tune the learning rate a little bit; but it's very sensitive; larger learning rates might cause the loss to blow.

**Comment by midskavid** [comment]
Thanks!

**Comment by zshyang** [comment]
Hi,
Do you know how to download the dataset and unzip it from the link below?
https://drive.google.com/open?id=131dH36qXCabym1JjSmEpSQZg4dmZVQid

**Comment by zshyang** [comment]
Hi,
Do you know how to use their provided YAML files and pretrained weights downloaded from here to do the evaluation?
To my understanding, I should use the downloaded meta folder. Then use the provide pretrained weights.
The command I use is 
```
python entrypoint_eval.py --name xxx --options experiments/default/tensorflow.yml --checkpoint datasets/data/pretrained/vgg16-p2m.pth 
```
But it seems that it only generated some meaningless meshes. And the score is really low.
I am not sure what I did wrong.
![Screenshot from 2020-09-22 02-08-41](https://user-images.githubusercontent.com/57812785/93863819-de0eac00-fc78-11ea-91b5-0235f5b4487b.png)
